pkg/accprovisioninput/apis/aci.ctrl/v1alpha1/types.go

```diff
@@ -0,0 +1,126 @@
+package v1alpha1
```

Comment:
Few of the configs which I pointed out in the acc-provision patch are also missing here...